client/editor.js

```diff
@@ -35,3 +35,13 @@ addFilter(
 	'crowdsignal-forms/with-fixed-position-control',
 	withFixedPositionControl
 );
+addFilter(
```

Comment:
Nit: Because this is directly related to the feedback block, maybe put it inside `client/blocks/feedback/index.js` ?